demos/kitchen_sink/Known issues.md

```diff
@@ -0,0 +1,6 @@
+# Demo KivyMD
+
+## Known issues
+- Have no limit minium widget
+- Text box don't show cursor
+- Dropdown Menu don't have scroll layer
```

Comment:
Why would we move this and why in this PR?

Reply:
I just added it to note the problems when I encountered using this module. Should i remove this ?

Reply:
Yes even though that could be a good idea sometimes, I would prefer to stick to the issue tracker for now

Reply:
OK, I will add that later